### PR TITLE
AE : Provide direct link for Ajman

### PR DESCRIPTION
**Pull request by BellezaEmporium**
seems to work without tokens.

### CODE REVIEW

File: streams/ae.m3u [hunk 1]
```diff
@@ -13,8 +13,8 @@ https://admdn5.cdn.mangomolo.com/adsports2/smil:adsports2.stream.smil/playlist.m
 https://admdn3.cdn.mangomolo.com/adsports3/smil:adsports3.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="AbuDhabiSports4.ae" status="blocked",Abu Dhabi Sports 4 (1080p) [Geo-blocked]
 https://admdn4ta.cdn.mgmlcdn.com/adsports4/smil:adsports4.stream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="AjmanTV.ae" status="error",Ajman TV
-https://q-s-l4n3sh4d0w.onrender.com/iptv-query?provider=dacast&streaming-ip=https://playback.dacast.com/content/access?contentId=116648_c_480929
+#EXTINF:-1 tvg-id="AjmanTV.ae" status="error",Ajman TV [Not 24/7]
+https://dacastmmd.mmdlive.lldns.net/dacastmmd/d3e9fc3b874a46159ce3724d802e3f8d/playlist.m3u8
```
Comment (BountyChocolate123456) [inline streams/ae.m3u:17]: Doesn't work for me

Comment (BountyChocolate123456) [reply streams/ae.m3u:17]: Maybe because https://www.ajmantv.com/ is not streaming?

Comment (BountyChocolate123456) [reply streams/ae.m3u:17]: works now. thanks

**Review by ghost — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Ajman has got another name, as a sort of "call sign" when streaming, being "UAE Television Ajman"

**Review by freearhey — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**